create/mysql.go

```diff
@@ -0,0 +1,94 @@
+package create
+
+import (
+	"context"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+
+	runtimev1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	infra "github.com/ninech/apis/infrastructure/v1alpha1"
+	meta "github.com/ninech/apis/meta/v1alpha1"
+	storage "github.com/ninech/apis/storage/v1alpha1"
+	"github.com/ninech/nctl/api"
+)
+
+type mySQLCmd struct {
+	Name                 string                                 `arg:"" default:"" help:"Name of the MySQL instance. A random name is generated if omitted."`
+	Location             string                                 `default:"nine-cz41" help:"Location where the MySQL instance is created."`
+	MachineType          infra.MachineType                      `help:"MachineType defines the sizing for a particular MySQL instance." placeholder:"nine-standard-1" default:"nine-standard-1"`
+	AllowedCidrs         []storage.IPv4CIDR                     `help:"AllowedCIDRs specify the allowed IP addresses, connecting to the instance." placeholder:"0.0.0.0/0"`
+	SSHKeys              []storage.SSHKey                       `help:"SSHKeys contains a list of SSH public keys, allowed to connect to the db server, in order to up-/download and directly restore database backups."`
+	SQLMode              *[]storage.MySQLMode                   `help:"SQLMode configures the sql_mode setting. Modes affect the SQL syntax MySQL supports and the data validation checks it performs."`
+	CharacterSet         storage.MySQLCharacterSet              `embed:"" prefix:"character-set-" help:"CharacterSet configures the character_set_server and collation_server variables."`
```

Comment:
This is a struct. How do I define it on the command line?
https://github.com/ninech/apis/blob/b8ea3d333f9d225a3c979a78ee8c91511784060c/storage/v1alpha1/types.go#L417

Reply:
Oh I see, the CLI library seems to be intelligent about it, but there is no help text:

```
   --sql-mode=SQL-MODE                                          SQLMode configures the sql_mode setting. Modes affect the SQL syntax MySQL supports and the data validation
                                                                   checks it performs.
      --character-set-name=STRING
      --character-set-collation=STRING
      --long-query-time=LONG-QUERY-TIME                            LongQueryTime configures the long_query_time variable. If a query takes longer than this many seconds, the the
                                                                   query is logged to the slow query log file.
```

So I'd say it makes sense to explicitly define the two?

Reply:
That's what the "embed" and "prefix" is for:
```
      --character-set-name=STRING
      --character-set-collation=STRING
```